python/ray/util/collective/util.py

```diff
@@ -40,3 +40,29 @@ def get_id(self):
             logger.warning("The NCCL ID has not been "
                            "set yet for store {}.".format(self.name))
         return self.nccl_id
+
+
+@ray.remote
+class Info:
+    """
+    Store the group information created via `declare_collective_group`.
```

Comment:
Should be on first line:
```suggestion
    """Store group information created via `declare_collective_group`.
```